SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -71,7 +71,10 @@ bool USpatialStatics::IsSpatialOffloadingEnabled(const UWorld* World)
 	if (World != nullptr)
 	{
 		const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings());
-		return IsSpatialNetworkingEnabled() && WorldSettings->WorkerLayers.Num() > 0;
+		if (WorldSettings != nullptr)
```

Comment:
NIT, we'd normally combine as 

`if (const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings()))`

Reply:
So I've updated the code, because now isn't the time for style discussions, but I find it really odd that if (WorldSettings) is against style and has to be if (WorldSettings != nullptr) but we're fine with if (const WorldSettings = ...).

Reply:
Yep I agree there's imbalance, currently the this topic is detailed here https://documentation.improbable.io/gdk-for-unreal/docs/coding-standards